src/components/PageContent/Billboard/Billboard.jsx

```diff
@@ -47,7 +47,7 @@ const Billboard = ({
 Billboard.displayName = "Billboard";
 Billboard.propTypes = {
   /** Optional image to display */
-  image: PropTypes.oneOfType(PropTypes.element),
+  image: PropTypes.oneOfType([PropTypes.element]),
```

Comment:
I'm not sure why these were `oneOfType` to begin with if we only accept one type.